Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.5
+
+RUN git clone https://github.com/TeamHG-Memex/autologin.git /opt/autologin
```

Comment:
Yeah, agreed. What is the recommended way to copy local files into the container without copying any undesired files (``*.pyc``, ``__pycache__``, etc.)?
I can think of two options: carefully ``.dockerignore`` all undesired files, or do ``python setup.py sdist`` and then extract the archive. But both these ways look a bit strange to me.